pages/mine/index.tsx

```diff
@@ -11,77 +11,69 @@ import { styled } from 'stitches.config';
 import { Fragment } from 'react';
 import InvitationButton from '@components/page/groupList/Card/InvitationButton';
 import Status from '@components/page/groupList/Card/Status';
-import { useQueryString } from '@hooks/queryString';
+import useSessionStorage from '@hooks/useSessionStorage';
+import { useGroupListOfApplied, useGroupListOMine } from 'src/api/user/hooks';
+import EmptyView from '@components/page/groupList/EmptyView';
+import { SSRSafeSuspense } from '@components/util/SSRSafeSuspense';
 
 const enum GroupType {
   MADE,
   APPLIED,
 }
 
 const MinePage: NextPage = () => {
-  const { value: selectedGroupType, setValue: setSelectedGroupType } =
-    useQueryString('groupType', String(GroupType.MADE));
+  const [selectedGroupType, setSelectedGroupType] = useSessionStorage(
+    'groupType',
+    GroupType.MADE as number
+  );
```

Comment:
```suggestion
  const [selectedGroupType, setSelectedGroupType] = useSessionStorage<GroupType>(
    'groupType',
    GroupType.MADE
  );
```

as number 지우고 이렇게 사용하는 게 좋을 것 같습니다!

Reply:
저게  Tab.Group 컴포넌트에 onChange 랑 setSelectedGroupType 이 파라미터 받는게 number랑 GroupType으로 달라서 as number로 적어줬던것 같은데
다시 지워보니 타입 에러가 안나네? 왜징? 
![스크린샷 2022-11-17 오후 6 21 51](https://user-images.githubusercontent.com/26538967/202406771-06eb74f8-28b8-4015-ab1f-f0cd823acfa8.png)
![스크린샷 2022-11-17 오후 6 22 01](https://user-images.githubusercontent.com/26538967/202406805-0e5882f8-b26f-4d02-b075-d11aafa0563e.png)